modules/picker/main/index.coffee

```diff
@@ -34,12 +34,16 @@ class Picker extends hx.EventEmitter
       renderer: undefined
       value: undefined
       disabled: false
+      fullWidth: false
     }, options
 
     hx.component.register(selector, this)
 
     @selection = hx.select(selector)
 
+    if resolvedOptions.fullWidth
+      @selection.classed('hx-picker-full-width', true)
```

Comment:
You could convert this to `@selection.classed('hx-picker-full-width', resolvedOptions.fullWidth)` ;)